.github/workflows/ci.yml

```diff
@@ -23,21 +23,35 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         scala: [2.12.15, 2.11.12, 2.13.8]
-        java: [temurin@11]
+        java: [temurin@8, temurin@11, temurin@17]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout current branch (full)
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
+      - name: Setup Java (temurin@8)
+        if: matrix.java == 'temurin@8'
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 8
+
```

Comment:
Given that Java 8 was eol 2019 I think we can skip that, adding 17 is good though.

```suggestion
```